sdrbase/util/aprs.cpp

```diff
@@ -16,10 +16,25 @@
 ///////////////////////////////////////////////////////////////////////////////////
 
 #include <QRegExp>
+#include <QRegularExpression>
 #include <QStringList>
 #include <QDateTime>
 
 #include "aprs.h"
+#include "util/units.h"
+
+inline bool inRange(unsigned low, unsigned high, unsigned x)
+{
+    return (low <= x && x <= high);
+}
+
+// Required for Mic-E Decoding
+inline int charToIntAscii(QString&s, int idx)
+{
```

Comment:
The other one returned `0` for a space, which breaks Mic-E course and speed use the ASCII range 28 to 127, which includes space (dec 32) and thus I rewrote to be more faithful.

I'm happy to rewrite the other one and have it return correctly and fix the space vs 0 issue where needed... otherwise this works.